### ISSUE TITLE
TypeError: keygrip.decrypt is not a function

### ISSUE THREAD

**Issue by skaterdav85** [adonisjs/adonis-framework#335]
Hi, I just set up the framework and created a model called User. I already have a table called `users`. As soon as I did the following in a controller method and invoked the route that triggered this code:

```
 const users = yield User.all()
response.json(user)
```

I got the following error:

```
TypeError: keygrip.decrypt is not a function
    at Object.Cookie._decryptCookie (/Users/dtang/Sites/side-projects/wedding-api/node_modules/node-cookie/src/Cookie/index.js:97:29)
    at /Users/dtang/Sites/side-projects/wedding-api/node_modules/node-cookie/src/Cookie/index.js:73:38
    at Array.forEach (native)
    at Object.Cookie.parse (/Users/dtang/Sites/side-projects/wedding-api/node_modules/node-cookie/src/Cookie/index.js:71:31)
    at CookieManager.read (/Users/dtang/Sites/side-projects/wedding-api/node_modules/adonis-framework/src/Session/CookieManager.js:63:23)
    at Session.getSessionId (/Users/dtang/Sites/side-projects/wedding-api/node_modules/adonis-framework/src/Session/index.js:122:42)
    at Session.get (/Users/dtang/Sites/side-projects/wedding-api/node_modules/adonis-framework/src/Session/index.js:255:61)
    at next (native)
    at onFulfilled (/Users/dtang/Sites/side-projects/wedding-api/node_modules/co/index.js:65:19)
    at /Users/dtang/Sites/side-projects/wedding-api/node_modules/co/index.js:54:5
```

Just wondering if you knew what I might be missing. If not, I'll keep digging. Thanks!

Node version 6.9.1 via NVM.

**Comment by thetutlage** [comment]
Seems to be like some weird install issue by npm. You can please remove and reinstall all dependencies?

**Comment by thetutlage** [comment]
Closing since no activity from reporter and does not seems to be bug either

**Comment by danieldecsi** [comment]
Hi,

I got the same error as @skaterdav85. I tried to remove and reinstall all the dependencies, it didn't help. However I figured out that this happens only when using Yarn, everything is fine after running npm install. While checking the version of Yarn I realised that I'm using an outdated version of Yarn (0.16.1). After upgrading it to latest version (0.19.1) and doing a clean install (rm -rf node_modules && rm yarn.lock && yarn cache clean && yarn) the problem is gone.
